docs/docs/concepts/jobs.md

```diff
@@ -3,7 +3,7 @@ title: Jobs
 sidebar_position: 6
 ---
 
-Jobs can be handled with [dm-job](https://github.com/equinor/dm-job).
+In the development framework, a "job" is a python script that will run and produce some result. Often, the python script will save the result of a job to an entity stored in DMSS. Jobs can be handled with [dm-job](https://github.com/equinor/dm-job).
```

Comment:
The formulation "python script" makes me think of a python file, not inline python code in a class. Perhaps rephrase to 
```suggestion
In the development framework, a "job" is a piece of python code that will run and produce some result. Often, the job will save the result to an entity stored in DMSS. Jobs can be handled with [dm-job](https://github.com/equinor/dm-job).
```